port_ocean/cli/commands/sail.py

```diff
@@ -52,4 +63,9 @@ def sail(
     print_logo()
 
     console.print("Setting sail... ⛵️⚓️⛵️⚓️ All hands on deck! ⚓️")
+
+    if once:
+        console.print("Setting event listener to IMMEDIATE")
+        os.environ["OCEAN__EVENT_LISTENER"] = '{"type": "IMMEDIATE"}'
```

Comment:
Can u pass it to the run instead of setting environs?

Reply:
it is problematic as currently the config loader knows how to handle envs and file, adding also parameters can make it less readable and is probably an over kill.